File: 240405/네 번째 문자/4th-letter.py
n,c=input().split()
n=int(n)
cnt=0
lists=[]
for _ in range(n):
    str=input()
    if(str[3]==c):
        lists.append(str)
        cnt+=1
print(cnt)
for i in range(cnt):
    print(lists[i])

File: 240905/네 번째 문자/4th-letter.kt
fun main()
{
    var (n,a) = readLine()!!.split(" ")
    var answer=""
    var cnt=0
    for(i in 1..n.toInt()) {
        val str = readLine()!!
        if (str[3] == a[0]) {
            if (cnt != 0) answer += "\n"
            answer += str
            cnt++
        }
    }
    println(cnt)
    print(answer)
}